app.js

```diff
@@ -189,6 +189,15 @@ exampleTemplatesApp.get(/^([^.]+)$/, (req, res, next) => {
 
 app.use('/prototype-admin', prototypeAdminRoutes);
 
+if (process.env.NODE_ENV === 'production') {
+  app.use((req, res, next) => {
+    // Set Strict-Transport-Security header to
+    // ensure that browsers only use HTTPS
+    res.setHeader('Strict-Transport-Security', 'max-age=31536000; preload');
```

Comment:
Can I check if [`includeSubDomains` is mandatory](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#preload) when `preload` is used?

```suggestion
    res.setHeader('Strict-Transport-Security', 'max-age=31536000; includeSubDomains; preload');
```

Reply:
Ah good point!